example_notebooks/5_ALS_hd_demo.ipynb

```diff
@@ -0,0 +1,365 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "61f605d4",
+   "metadata": {},
+   "source": [
+    "# ALS HD demo scene\n",
+    "\n",
+    "Notebook creator:\n",
```

Comment:
Do you want to add your name or initials here (or ours) plus the year?
Lukas put `Notebook: Lukas Winiwarter, 2022`